docs/requirements.md

```diff
@@ -22,7 +22,7 @@ The cluster running Codacy must satisfy the following requirements:
 
 -   The infrastructure hosting the cluster must be provisioned with the hardware and networking requirements described below
 -   The orchestration platform managing the cluster must be one of:
-    -   [Kubernetes](https://kubernetes.io/) **version 1.22.\*** to **1.23.\*** (1.23 recommended)
+    -   [Kubernetes](https://kubernetes.io/) **version 1.22.\*** to **1.24.\*** (1.23 recommended)
```

Comment:
I'm leaving 1.23 as the recommended version since it will also be the recommended version for SH12 (see [Slack message](https://codacy.slack.com/archives/CSSSHAK9N/p1687434639879449?thread_ts=1687166816.965619&cid=CSSSHAK9N) for the rationale).

Reply:
Thank you - This is perfect.